WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -695,5 +816,14 @@ private extension ProductSelectorViewModel {
                 "Filter (%ld)",
                 comment: "Title of the button to filter products with filters applied on the Select Product screen"
         )
+        static let popularProductsSectionTitle = "Popular"
+        static let lastSoldProductsSectionTitle = "Last Sold"
+        static let productsSectionTitle = "Products"
```

Comment:
We could declare these as `NSLocalizedString`:
```suggestion
        static let popularProductsSectionTitle = NSLocalizedString("Popular", comment: "Title of the Popular Products section in the Select Product screen")
        static let lastSoldProductsSectionTitle = NSLocalizedString("Last Sold", comment: "Title of the Last Sold section in the Select Product screen")
        static let productsSectionTitle = NSLocalizedString("Products", comment: "Title of the Products section in the Select Product screen")
```

Reply:
Done in 7826df7ceeedb08a255a5a95716d38da7e606e38